src/protocol.cpp

```diff
@@ -47,12 +47,6 @@ const char *GETSPORKS="getsporks";
 const char *MASTERNODEPAYMENTVOTE="mnw";
 const char *MASTERNODEPAYMENTBLOCK="mnwb";
 const char *MASTERNODEPAYMENTSYNC="mnget";
-const char *MNBUDGETSYNC="mnvs"; // deprecated since 12.1
-const char *MNBUDGETVOTE="mvote"; // deprecated since 12.1
-const char *MNBUDGETPROPOSAL="mprop"; // deprecated since 12.1
-const char *MNBUDGETFINAL="fbs"; // deprecated since 12.1
-const char *MNBUDGETFINALVOTE="fbvote"; // deprecated since 12.1
-const char *MNQUORUM="mn quorum"; // not implemented
```

Comment:
Can't drop this, see below

Reply:
Yeah, I didn't realize they needed to be in-sync (esp these `NsgMsgType` constants here, since those aren't an enum and just directly assigned values). I mean, in theory we could replace these with a single `const char *NOTIMPLEMENTED = "notimplemented";`, right?

But for the sake of not making any functional changes I'll revert this commit.

Reply:
Yep, `NOTIMPLEMENTED` would work I guess.